bindings.c

```diff
@@ -4477,6 +4479,19 @@ static bool permute_root(void)
 	return true;
 }
 
+static int preserve_ns(int pid)
+{
+	int ret;
+	size_t len = 5 /* /proc */ + 21 /* /int_as_str */ + 7 /* /ns/mnt */ + 1 /* \0 */;
```

Comment:
Any reason not to do "sizeof("/proc") + sizeof("/ns/mnt") + ...?

Reply:
No, can do.